### PR TITLE
Make sure curl is available in final Docker image

### PR DESCRIPTION
**Pull request by duyanton**
### Motivation / Background

When deploy an image with Mrsk, the container needs to have `curl` available to run healthcheck.

See: https://github.com/mrsked/mrsk/pull/219

### Detail

This PR will make `curl` available in the final image.

### Checklist

Before submitting the PR make sure the following are checked:

* [x] This Pull Request is related to one change. Changes that are unrelated should be opened in separate PRs.
* [x] Commit message has a detailed description of what changed and why. If this PR fixes a related issue include it in the commit message. Ex: `[Fix #issue-number]`
* [x] Tests are added or updated if you fix a bug or add a feature.
* [ ] CHANGELOG files are updated for the changed libraries if there is a behavior change or additional feature. Minor bug fixes and documentation changes should not be included.

### CODE REVIEW

File: railties/lib/rails/generators/rails/app/templates/Dockerfile.tt [hunk 1]
```diff
@@ -70,7 +70,7 @@ FROM base
 <% unless dockerfile_deploy_packages.empty? -%>
 # Install packages needed for deployment
 RUN apt-get update -qq && \
-    apt-get install --no-install-recommends -y <%= dockerfile_deploy_packages.join(" ") %> && \
+    apt-get install --no-install-recommends -y curl <%= dockerfile_deploy_packages.join(" ") %> && \
```
Comment (guilleiguaran) [inline railties/lib/rails/generators/rails/app/templates/Dockerfile.tt:73]: Perhaps curl should be added to the [dockerfile_deploy_packages](https://github.com/rails/rails/blob/1545a8d74b08ecd407afbb9467c202f279da377b/railties/lib/rails/generators/app_base.rb#L558) list?

Comment (duyanton) [reply railties/lib/rails/generators/rails/app/templates/Dockerfile.tt]: Updated, thanks.

**Comment by guilleiguaran** [comment]
Merged it https://github.com/rails/rails/commit/c6d5baaf8ddee8a65d62f38eb7a82aa7e5fde994

**Comment by 34code** [comment]
yay!

**Pull request closed without merging**